README.md

```diff
@@ -4,7 +4,7 @@
 
 - [learn more about policies](https://docs.spacelift.io/concepts/policy) and create one from scratch;
 - create some resources on a real public cloud;
-- [connect your Slack workspace](https://docs.spacelift.io/integrations/slack);
+- [connect your Slack workspace](https://docs.spacelift.io/integrations/chatops/slack.html);
 - learn about our native [cloud integrations](https://docs.spacelift.io/integrations/cloud-providers);
 - set up [SSO for your organization](https://docs.spacelift.io/integrations/single-sign-on);
-- start a [Spacelift agent](https://docs.spacelift.io/concepts/private-worker-pools) in your own infrastructure (yes, it will run on your laptop, too);
+- start a [Spacelift agent](https://docs.spacelift.io/concepts/worker-pools#launch-private-worker-pool) in your own infrastructure (yes, it will run on your laptop, too);
```

Comment:
The URL should be `https://docs.spacelift.io/concepts/worker-pools#launch-worker-pool` so if you can update this that would be great!